src/main.rs

```diff
@@ -249,20 +249,16 @@ fn main() {
     let args = Cli::parse();
 
     // Determine the usage of styles
-    #[cfg(target_os = "windows")]
-    let ansi_support = yansi::Paint::enable_windows_ascii();
-    #[cfg(not(target_os = "windows"))]
-    let ansi_support = true;
     let enable_styles = match args.color.unwrap_or_default() {
         // Attempt to use styling if instructed
-        ColorOptions::Always => true,
+        ColorOptions::Always => {
+            yansi::enable(); // disable yansi's automatic detection for ANSI support on Windows
```

Comment:
@nc7s Sorry to bother you again, but how do you feel about this change I added? It should now be exactly the previous behavior right?

Reply:
I don't really know enough to judge this, from a glance LGTM?